.travis.yml

```diff
@@ -0,0 +1,26 @@
+language: php
+
+matrix:
+    include:
+        - php: 7.3
+
+install:
+    - composer install
+
+script:
+    # test that website is generated
+    - vendor/bin/statie generate source
+
+# see docs: https://www.statie.org/docs/github-pages/
+deploy:
+    provider: pages
+    skip_cleanup: true
+    github_token: $GITHUB_TOKEN
```

Comment:
You still need to add GITHUB_TOKEN env to Travis for this repository.

See https://www.statie.org/docs/github-pages/

Reply:
If you've added the token, then:

- create `gh-pages` branch with whatever content, it will be overriden by Travis
- pick it as source for Github pagaes - https://github.com/janmikes/janmikes.cz/settings
- merge this
- it should deploy in 2-5 minutes
- if not, ping me